static/app/components/profiling/flamegraph/flamegraphZoomView.tsx

```diff
@@ -870,7 +870,7 @@ function handleFlamegraphKeyboardNavigation(
     direction = direction === 'up' ? 'down' : 'up';
   }
   const nextSelection = selectNearestFrame(currentFrame, direction);
-  if (!nextSelection) {
+  if (!nextSelection || nextSelection === currentFrame) {
```

Comment:
I dont think we need !nextSelection since selectNearestFrame always returns a frame

Reply:
😅  lets try this again